CHANGELOG.md

```diff
@@ -4,6 +4,52 @@ This document includes a curated changelog for each release. We also publish a c
 a [GitHub release](https://github.com/nginxinc/nginx-gateway-fabric/releases), which, by contrast, is auto-generated
 and includes links to all PRs that went into the release.
 
+## Release 1.1.0
+
+*December 13, 2023*
+
+This is the official v1.1.0 release of NGINX Gateway Fabric.
+
+FEATURES:
+
+- Update to v1.0.0 of the Gateway API. [PR-1250](https://github.com/nginxinc/nginx-gateway-fabric/pull/1250)
+- Set the SupportedVersion Condition on GatewayClass. [PR-1301](https://github.com/nginxinc/nginx-gateway-fabric/pull/1301)
+
+BUG FIXES:
+
+- Merge HTTPRoute conditions from all Gateway controllers. [PR-1220](https://github.com/nginxinc/nginx-gateway-fabric/pull/1220)
+- Validate header names and report validation errors in the HTTPRoute status. [PR-1239](https://github.com/nginxinc/nginx-gateway-fabric/pull/1239)
+- Remove usage info from log output. [PR-1242](https://github.com/nginxinc/nginx-gateway-fabric/pull/1242)
+- Fix attaching of HTTPRoutes to Gateway Listeners. [PR-1275](https://github.com/nginxinc/nginx-gateway-fabric/pull/1275)
```

Comment:
This was actually for 'invalid' HTTPRoutes and Listeners. Might we want to change this note to reflect that? As it reads now, it sounds as if you couldn't attach httproutes at all.

Reply:
Reworded this. Let me know what you think @mpstefan